Common/Algorithm/Framework/Alphas/Analysis/InsightAnalysisContext.cs

```diff
@@ -166,6 +166,10 @@ public bool ShouldAnalyze(InsightScoreType scoreType)
             {
                 return Insight.Magnitude.HasValue;
             }
+            else if (scoreType == InsightScoreType.Direction)
```

Comment:
Approving and merging for now -- but just a caveat for future; once we add volatility predictions this expression need to allow flat predictions to be scored as vol would be predicting it will continue within (+-)Magnitude-range.